config/locales/en.yml

```diff
@@ -407,6 +408,11 @@ en:
         csv_hint: A comma-separated values (CSV) lists all the answers in plain text, separated by commas. It can be imported into spreadsheets and other applications.
         csv_sample: See sample CSV
         csv_checkbox: Send a separate email with answers in a CSV
+        from_address:
+          default: You need to set and validate a team email address.
+          pending: You need to validate this email address.
+          pending_link: 'Get validation link'
+          link: Change 'from' address
```

Comment:
This should have 'curly' quotes not straight ones.  If you have a look elswhere in the yml file you'll find some you can copy.

Reply:
done